attack_utilities/estimator_template.py

```diff
@@ -0,0 +1,86 @@
+'''
```

Comment:
I think this probably needs better commenting / docs.
Do we actually use it though? I thought these classes had been superseded by the safe wrappers?